blog/poc3.md

```diff
@@ -1,120 +1,64 @@
 
-Here is how it works: a publisher creates a mutable resource at time `t` for the name "my daily tweets", that she intends to update every day once (note this in itself is not a promise). By hashing together `ID := SHA3(publisher_pubkey, name, start, period)`, we obtain a so called metadata chunk.
-A publisher creates a new version of a mutable resource by serialising `(name, start, period, version)` and signing it.
-This simple system is surprisingly powerful and is envisioned to be at the core of implementing group chat, shared FUSE state, and [poor man's consensus](https://www.sharelatex.com/1452913241cqmzrpfpjkym) (soft channel deposit).
+Swarm POC3 adds another layer in the form of [*Mutable Resource Updates (MRU)*](http://swarm-guide.readthedocs.io/en/latest/usage.html#Mutable). These allow off-chain updates of content associated with an address at a much faster pace than ENS updates on the blockchain could support, and without incurring the cost of on-chain transactions.
 
-MRU is an experimental feature in current POC3 testnet and is still  undergoing changes.
+MRU is an experimental feature in current POC3 testnet and is still undergoing changes.
 
 ## FUSE support
 
-We mentioned that thanks to manifests, directory trees and collections represented by manifests can be mapped uniquely. This does not only allow upload and download into directory but also thanks to [FUSE](http://swarm-guide.readthedocs.io/en/latest/usage.html#FUSE), enables users to mount a Swarm manifest as a local file system (only available on Linux and Mac).
-Due to the fact that Swarm retrieval is on-demand, Swarm FUSE mounts are readable with no latency. Swarm FUSE supports file system write operations (creating, appending, updating files) by maintaining the collection's Swarm hash as an internal state.
-Combining FUSE with MRU, on each change we can issue a mutable resource update, this makes it possible to think of FUSE to sync your entire home folder between devices, virtually serving as the backend to a decentralised drop box.
+[FUSE](http://swarm-guide.readthedocs.io/en/latest/usage.html#FUSE), enables users to integrate Swarm data directly into their local file systems (only on Linux and Mac). Using this system, users can "mount a Swarm manifest" as if it were a regular directory. It supports file system read and write operations, in which all content is automatically synced with the Swarm.
+In future, combining FUSE with Swarm Mutable Resources, it should be possible sync your entire home folder between devices - the backend to a decentralised storage with dropbox-like functionality.
```

Comment:
it should be possible **to** sync

Reply:
your entire home folder or just "any folder"?

Reply:
yes its better to say 

it should be possible to sync, for instance, your home folder between ...